include/bluetooth/hci.h

```diff
@@ -1258,6 +1322,213 @@ struct bt_hci_cp_le_set_privacy_mode {
 	uint8_t         mode;
 } __packed;
 
+#define BT_HCI_OP_LE_READ_BUFFER_SIZE_V2        BT_OP(BT_OGF_LE, 0x60)
+struct bt_hci_rp_le_read_buffer_size_v2 {
```

Comment:
Why no comments?

Reply:
What comments are you expecting? There is no comments to any HCI commands in this files since they are a direct map to the specification.